addons/crm/wizard/crm_lead_pls_update_views.xml

```diff
@@ -12,7 +12,7 @@
                     <field name="pls_fields" widget="many2many_tags" placeholder="Extra fields..."/>
                 </p>
                 <p>
-                    Consider leads created as of the: <field name="pls_start_date"/>
+                    Consider leads created as of the: <field name="pls_start_date" class="border-bottom"/>
```

Comment:
What about using the `o_field_highlight` class ? :slightly_smiling_face:

Reply:
@amdi-odoo Hello, good catch!

I was looking for such a class. The feeling is different, but its purpose seems to fit perfectly here. Let us go with it then :)

Cheers!

Reply:
Well, I just tested it because I was curious about what you meant by `the feeling is different` and the field doesn't seem to be always underlined ? :thinking: Or is it just me ?
![image](https://user-images.githubusercontent.com/107679666/223393240-d0beb98f-3e96-4700-bfda-2aefd736467d.png)

Another css rule on the input seems to override the `o_field_highlight` class behavior :
![image](https://user-images.githubusercontent.com/107679666/223392387-306cdd05-309b-45c1-ae03-ef9743ac4a42.png)
And removing this rule seems to make the `o_field_highlight` class work ?
![image](https://user-images.githubusercontent.com/107679666/223393480-3a8bf327-b3f3-4c1a-8381-dbbbd189ef04.png)

Reply:
Have you tested the latest runbot? I don't seem to have that issue :p it is always underlined. What I meant by "the feeling is different" is that even though we are not hovering that field, it is boldly underlined. With "border-bottom", it was first light grey then underlined when hovering.

Reply:
Oh my bad sorry! Forgot to update my locals :joy: Nice then! :slightly_smiling_face: Thank you!